api/toko-rest-api.yaml

```diff
@@ -227,6 +227,68 @@ paths:
                 $ref: '#/components/schemas/User'
         401:
           description: The request requires user authentication
+  /users/{id}:
+    get:
+      tags:
+        - Users
+      summary: Get user by ID
+      description: >
+        This method returns user with given ID
+      operationId: getUser
+      parameters:
+        - $ref: '#/components/parameters/EntityId'
+      responses:
+        200:
+          description:
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/User'
+        404:
+          description: User with a given ID has not beed found
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        410:
+          description: The user has already been deleted
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+    patch:
+      tags:
+        - Users
+      summary: Updated user
```

Comment:
This is summary for operation itself, so I think it would be clearer to say `Update user`